### PR TITLE
Seyi - styled communityDashboard Link

### LINKED ISSUE #106: Seyi- Styling Community direct Link
community site

### PR DESCRIPTION
**Pull request by Sheyitofunmi**
## What type of PR is this? (check all applicable)

- [ ] Feature
- [ ] Refactor
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [x] Other (please specify here: <type here>)
  
  
## Description of what this Pull Request does
styled communityDashboard Link

## Related Tickets & Documents
- Closes #106 
- Related Issues: (enter ticket numbers here)
  
## QA Instructions, Screenshots, Recordings
pic

## Added/updated tests?
- [ ] Yes
- [ ] Not Yet, and here is why: __explain here__
- [ ] I need help with tests
- [x] No, and here is why: __explain here__

### CODE REVIEW

**Review by jjleigh — APPROVED** [review]
LGTM

File: src/pages/singleListing.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
+import Link from "next/link";
+import NavBar from "@/components/Header/NavBar";
+import CustomizableAvatar from "@/components/common/customavatar/CustomizableAvatar";
+import CustomizableButton from "@/components/common/CustomizableButton";
+import FooterBar from "@/components/common/FooterBar";
+import styles from "../styles/singleListing.module.css";
+
+
+function SingleListing(className, text) {
```
Comment (msarit) [inline src/pages/singleListing.js:9]: @Sheyitofunmi there is this file `src/pages/[community_id].js` that I believe is the SingleListing page. In this PR you are creating a new but unnecessary one. Please make your changes to the `src/pages/[community_id].js` file and delete this file, thanks!

**Review by msarit — CHANGES_REQUESTED** [review]

File: src/pages/singleListing.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                
+            </div>
+        
+            
+            <FooterBar />
+        </>
+    )
+};
+
+export default SingleListing;
```
Comment (jjleigh) [inline src/pages/singleListing.js:79]: Add a new final line here

File: src/pages/singleListing.js (hunk 1 above)
Comment (Sheyitofunmi) [reply src/pages/singleListing.js:9]: alright ma, will make changes

File: src/pages/singleListing.js (hunk 2 above)
Comment (Sheyitofunmi) [reply src/pages/singleListing.js:79]: alright

File: src/pages/singleListing.js (hunk 1 above)
Comment (msarit) [reply src/pages/singleListing.js:9]: @Sheyitofunmi Please delete this file: src/pages/singleListing.js

**Review by msarit — CHANGES_REQUESTED** [review]

**Review by msarit — APPROVED** [review]

**Pull request merged**